app/controllers/reviews_controller.rb

```diff
@@ -43,6 +43,37 @@ def destroy
     redirect_to recipe
   end
 
+  def upvote
```

Comment:
This is non-blocking for the story, but I think this could much more easily be extracted into a `VoteController`.